WrfTrigger.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/python3
+
+# Download Rain cell files, download Mean-ref files.
```

Comment:
This scrip it Download Rain cell file, download mean files, and check. But I can't understand it's functionalities with the file name. 
Better to come up with new name, or divide into few scripts with single responsibilities